data/relation-budafok.yaml

```diff
@@ -551,9 +551,15 @@ filters:
     # 1: ez és a Nagytétényi út 2. közös telken van, házszám a Nagytétényi útról van csak.
     # 25/A: nincs ilyen, csak sima 25 van.
     # 27/A: nincs ilyen, csak sima 27 van.
+    # 49/A: nincs ilyen, csak sima 49 van.
+    # 59/A: TODO: kettő darab 59 van egymás mellett, nem egyértelmű, hogy melyik az 59/A.
+    # 65/A: nincs ilyen, csak sima 65 van.
+    # 4: a páros oldalon a 10 az első házszám.
+    # 6: a páros oldalon a 10 az első házszám.
+    # 40/2: TODO: kettő darab 40 van egymás mellett, nem egyértelmű, hogy melyik a 40/2.
     # 58: a páros oldalon az 52 az utolsó házszám.
     # 614: a páros oldalon az 52 az utolsó házszám.
```

Comment:
talán érdemes lenne range-et felvenni ilyen esetben (ha definit az utca vége), ha később jönnek nagyobb számok, akkor ne kellejen visszamenni

Reply:
Történt már olyan, hogy „meghosszabbodott” az utca újépítésű házakkal. Ha új adat lesz a referenciában, akkor azt a terepen is validálni kell. Nem olyan gyakori ez a fajta adathiba.